packages/api/src/external/ehr/athenahealth/command/process-patients-from-appointments.ts

```diff
@@ -186,12 +185,10 @@ async function getAppointments({
 }
 
 type GetAppointmentsFromApiParams = GetAppointmentsParams & {
-  tokenId: string;
   log: typeof console.log;
```

Comment:
FYI (no need to change here) - I added the type `LogFunction to `core/util/log` for accurately typing `out(...).log` and `out(...).debug`:

```
import { LogFunction } from "@metriport/core/util/log";
```